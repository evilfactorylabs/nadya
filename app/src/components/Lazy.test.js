import LazyNavbar from './Lazy'

afterAll(async () => {
  await cleanup()
})

describe('Fragments: Lazy.js', () => {
  test('It should render Lazy screen', () => {
    const { container } = render(<LazyNavbar component='./Navbar' />)

    expect(container).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
