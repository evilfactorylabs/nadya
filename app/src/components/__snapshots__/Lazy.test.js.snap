// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fragments: Lazy.js It should render Lazy screen 1`] = `
<div>
  <div
    class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
    role="progressbar"
    style="position: absolute; top: 0px; left: 0px; width: 100%;"
  >
    <div
      class="MuiLinearProgress-bar MuiLinearProgress-bar1Indeterminate MuiLinearProgress-barColorPrimary"
    />
    <div
      class="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
    />
  </div>
</div>
`;
